docs/source/projects.rst

```diff
@@ -375,18 +375,59 @@ in the Databricks docs
 of how to use the feature is as follows:
 
 1. Create a JSON file containing the
-`new cluster specification <https://docs.databricks.com/api/latest/jobs.html#jobsclusterspecnewcluster>`_
-for your run. For example:
+`new cluster specification <https://docs.databricks.com/dev-tools/api/latest/jobs.html#jobsclusterspecnewcluster>`_
+or `cluster specification <https://docs.databricks.com/dev-tools/api/latest/jobs.html#clusterspec>`_
+for your run. If you do not need to specify libraries installed on the Spark worker nodes, you can use the
+simpler "new cluster specification" format. For example:
 
   .. code-block:: json
 
     {
-      "spark_version": "5.5.x-scala2.11",
+      "spark_version": "6.4.x-scala2.11",
       "node_type_id": "i3.xlarge",
       "aws_attributes": {"availability": "ON_DEMAND"},
       "num_workers": 4
     }
 
+If you do need to install libraries on the worker, use the "cluster specification" format. For example:
+
+  .. code-block:: json
+
+    {
+      "new-cluster": {
```

Comment:
```suggestion
      "new_cluster": {
```

Reply:
Yep, true! Code and tests expect that too!